### PR TITLE
Add wasm modules

### PR DESCRIPTION
**Pull request by dsherret**

### CODE REVIEW

File: reference_gen/web-categories.json
Comment (dsherret) [inline reference_gen/web-categories.json:19]: https://github.com/denoland/deno/pull/26954/files

File: runtime/reference/wasm.md
Comment (bartlomieju) [inline runtime/reference/wasm.md:20]:
```suggestion
Starting in Deno 2.1, WebAssembly modules can be imported and their use type
```

**Review by bartlomieju — APPROVED** [review]

**Comment by dsherret** [comment]
Will wait on the release before merging in order to pick up https://github.com/denoland/deno/pull/26954/files

**Comment by bartlomieju** [comment]
@dsherret you can now merge it

**Pull request merged**
